reference-adaptations/Night%20Gear/header.sqe

```diff
@@ -1,4 +1,4 @@
 version=53;
 name="Night Gear";
 author="TomHighway";
-category="CNTO_Defaults_Adaptations";
+category="CNTOStandardFactionAdaptation";
```

Comment:
Maybe rename to "CNTO_Faction_Adaptations" to make it a little shorter and neater.

With or without _ up to you.